src/views/Home.vue

```diff
@@ -1,7 +1,13 @@
 <template>
   <div class="home">
-    <Title msg="Which Anime Character Are You?" />
-    <Button text="Start Quiz" />
+    <div class="figure">
```

Comment:
Lets name these using BEM. So home is the block, then you would have home__images & home__content instead of figure and section and nest them accordingly in the css